src/sysrepo_types.h

```diff
@@ -167,6 +167,16 @@ struct sr_error_info_s {
 
 typedef struct sr_error_info_s sr_error_info_t;
 
+/**
+ * @brief Error type as used in the NETCONF and RESTCONF protocols
+ */
+typedef enum {
```

Comment:
I would be OK with using a string for this thing if you prefer to bake less NETCONF/RESTCONF info into sysrepo.

Reply:
Yes, I think I would prefer that.